File: app/src/main/java/com/pluu/webtoon/di/NetworkComponent.java
package com.pluu.webtoon.di;

import com.pluu.support.impl.NetworkSupportApi;

import javax.inject.Singleton;

import dagger.Component;

/**
 * NetworkComponent
 * Created by PLUUSYSTEM-SURFACE on 2016-07-12.
 */
@Singleton
@Component(modules = {NetworkModule.class})
public interface NetworkComponent {

    void inject(NetworkSupportApi api);

}

File: app/src/main/java/com/pluu/webtoon/di/NetworkComponent.kt
package com.pluu.webtoon.di

import com.pluu.support.impl.NetworkSupportApi
import dagger.Component
import javax.inject.Singleton

/**
 * NetworkComponent
 * Created by pluu on 2017-04-30.
 */
@Singleton
@Component(modules = arrayOf(NetworkModule::class))
interface NetworkComponent {

    fun inject(api: NetworkSupportApi)

}
